MAUI/DataGrid/Grouping.md

```diff
@@ -0,0 +1,572 @@
+                        AllowGroupExpandCollapse="True"
+                        ItemsSource="{Binding Orders}"/>
+{% endhighlight %}
+{% highlight c# %}
+this.dataGrid.AutoExpandGroups = true;
+this.dataGrid.AllowGroupExpandCollapse = true;
+{% endhighlight %}
+{% endtabs %}
+
+## Expand or collapse the groups
+
+By default, the groups will be in expanded state. However expand or collapse a group in runtime by setting the [SfDataGrid.AllowGroupExpandCollapse]() to `true`.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfDataGrid  x:Name="dataGrid"
+                        AllowGroupExpandCollapse="True"
+                        ItemsSource="{Binding Orders}"/>
+                       
+{% endhighlight %}
+{% highlight c# %}
+this.dataGrid.AllowGroupExpandCollapse = true;
+{% endhighlight %}
+{% endtabs %}
+
+### Expand or collapse all the groups
+
+Programmatically expand or collapse all the groups at runtime using the [SfDataGrid.ExpandAllGroup]() and [SfDataGrid.CollapseAllGroup]() methods.
+
+{% tabs %}
+{% highlight c# %}
+this.dataGrid.ExpandAllGroup();
+this.dataGrid.CollapseAllGroup();
+{% endhighlight %}
+{% endtabs %}
+
+### Expand or collapse a specific group
+
+Expand or collapse specific group by using the [SfDataGrid.ExpandGroup]() and [SfDataGrid.CollapseGroup]() methods.
+
+
+{% tabs %}
+{% highlight c# %}
+var group = (dataGrid.View.Groups[0] as Group);
+this.dataGrid.ExpandGroup(group);
+this.dataGrid.CollapseGroup(group);
+{% endhighlight %}
+{% endtabs %}
+
+![DataGrid with group expanding and collapsing](Images\Grouping\Expand_Specific_Group.png)
+
+## Display based grouping using GroupMode property
+
+By default, column grouping occurs based on the value in the underlying collection thereby creating a new group for each new value of that column. However, a column can also be grouped based on the Display value by setting the [GridColumn.GroupMode]() property as `Display`. In the following code example, set the [GridColumn.Format](t) property as "#" which displays only the rounded off value in the [GridCell](): 
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:DataGridTextColumn HeaderText="Order ID" 
+                           MappingName="OrderID"
+                           GroupMode="Display"
+                           Format="#" />
+{% endhighlight %}
+{% highlight c# %}
+DataGridTextColumn orderID = new DataGridTextColumn();
+orderID.MappingName = "OrderID";
+orderID.GroupMode = Syncfusion.Maui.Data.DataReflectionMode.Display;
+orderID.Format = "#";
+{% endhighlight%} 
+{% endtabs %}
+
+Following screenshot shows the comparison between two Group modes. GroupMode.Value on the Top and GroupMode.Display on the Bottom:
+![DataGrid with group mode](Images\Grouping\Mode_Value.png)
+![DataGrid with group mode](Images\Grouping\Mode_Display.png)
+
+## Clearing or removing a group
+
+To clear applied grouping, remove the items from the `SfDataGrid.GroupColumnDescriptions` collection or clear the collection. 
```

Comment:
To clear the applied grouping, you can simply remove the items from the `SfDataGrid.GroupColumnDescriptions` collection or clear the collection.

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/d3795eab-5c8e-4cb8-9ac9-24a929d37552)